### ISSUE TITLE
libmwr includes are missing in installation directory

### ISSUE THREAD

**Issue by pipsoft** [machineware-gmbh/vcml#60]
Trying the example from "https://github.com/machineware-gmbh/vcml/blob/main/doc/session.md" and I stumbled that the constructor of vpserver would not match my usage. That's when I realized that there is an issue with the type "u16". Looking into types.h I then notice that <mwr.h> is giving me a file not found. A find . -name "mwr.h" in the installation dir of vcml gives me nothing, while that find in the build directory gives me ./build/mwr-src/include/mwr.h

**Comment by aut0** [comment]
Normally, if you just use cmake all vcml includes and vcml dependencies should be resolved automatically.
Can you please provide more detailed steps to reproduce your issue, e.g. how you are building your executable?

**Comment by pipsoft** [comment]
I started with a clean Ubuntu Virtual Box image. Had to do some apt install cmake etc. of course. Then followed instructions of https://github.com/machineware-gmbh/vcml which worked like a charm (thanks!). libelf-dev, libsdl2-dev and libvncserver-dev are mentioned there. Nothing about mwr. I did everything step by step from there till "sudo make install". The make test gives me 100% coverage.
find / -name "mwr.h" delivers only one result which is in my home /home/frankp/TMP/vcml/build/mwr-src/include/mwr.h
